extensions/amp-analytics/0.1/test/test-visibility-model.js

```diff
@@ -979,6 +979,22 @@ describes.sandboxed('VisibilityModel', {}, () => {
         totalVisibleTime: 1000,
       });
     });
+
+    it('should fire for 100 percent min and max visibility', () => {
+      const vh = new VisibilityModel({
+        continuousTimeMin: 1000,
```

Comment:
nit: can you remove the continuousTimeMin from the spec.
the test seems to be testing the timing, rather than the visible percentage.

Reply:
Good point. Done.